samples/azure-quantum/chemistry/QuantumPhaseEstimation/Molecule.ipynb

```diff
@@ -0,0 +1,1775 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simulate molecular ground state with quantum algorithms\n",
+    "\n",
+    "In this example, we'll simulate the molecular ground state of caffeine and hydrogen (H2) using Quantum Phase Estimation (QPE) and Variational Quantum Eigensolver (VQE). We'll show how many quantum resources (qubits, gates) the different approaches require and run the results on a quantum simulator.\n",
+    "\n",
+    "To read more about these two algorithms, take a look at our blog post on the Q# Dev blog: [Quantum developer tools for chemistry](https://devblogs.microsoft.com/qsharp/quantum-developer-tools-for-chemistry/)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/json": "{\"LastUpdated\":\"2021-05-20T17:37:40.5328178-07:00\",\"IsCompleted\":false,\"Description\":\"Adding package microsoft.quantum.chemistry.jupyter\",\"Subtask\":null}",
+      "text/plain": [
+       "Adding package microsoft.quantum.chemistry.jupyter."
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/json": "{\"LastUpdated\":\"2021-05-20T17:37:42.1312255-07:00\",\"IsCompleted\":false,\"Description\":\"Reloading workspace\",\"Subtask\":null}",
+      "text/plain": [
+       "Reloading workspace."
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import qsharp\n",
+    "from qdk.chemistry import Molecule\n",
+    "from qdk.chemistry.broombridge import load_and_encode\n",
+    "qsharp.reload()\n",
+    "from Microsoft.Quantum.Chemistry.QPE import GetEnergyQPE\n",
+    "from Microsoft.Quantum.Chemistry.VQE import GetEnergyVQE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Load molecule"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "caffeine = Molecule.from_xyz(\"../data/xyz/caffeine.xyz\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "cda7d8ae7cee43499ef2205cfb2183f9",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "JsmolWidget(layout=Layout(align_self='stretch', height='400px'))"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "caffeine"
```

Comment:
It looks like this cell output isn't captured in nbviewer or GitHub's notebook renderer...?

Reply:
That's because it's an interactive JS widget.